routes/cds-services.js

```diff
@@ -234,9 +237,7 @@ async function call(req, res, next) {
       console.log('Request context: ', req.body.context);
       console.log('searchURL ', searchURL);
 
-      const result = await client.request(searchURL, { pageLimit: 0, flat: true });
-      await addDiagnosticReportToBundle(result, bundle, client);
-
+      await client.request(searchURL, { pageLimit: 0, flat: true });
       let patientData = bundle.entry.map(b => b.resource);
```

Comment:
I believe we'll want to add back the `result` variable here. This result gets fed into `translateResponse()` below.

```suggestion
      const result = await client.request(searchURL, { pageLimit: 0, flat: true });
```

Reply:
Resolved in https://github.com/ccsm-cds-tools/ccsm-cds-service/pull/27/commits/d9cf17a17756059239410649dabdef8e7d7f81e9.